client/hack/README.md

```diff
@@ -8,7 +8,7 @@ This README documents:
 
 This is the script to update clientset/informers/listers and API deepcopy code using [code-generator](https://github.com/kubernetes/code-generator).
 
-Make sure to run this script after making changes to /client/apis/volumesnapshot/v1beta1/types.go.
+Make sure to run this script after making changes to /client/apis/volumesnapshot/v1/types.go.
```

Comment:
My understanding is that v1beta1/types.go still exists. This PR only add a v1/types.go into the repo is that right?

Reply:
Yes, but we no longer makes changes to v1beta1.